blackbox/docs/admin/auth/hba.txt

```diff
@@ -160,3 +159,32 @@ the the ``auth.host_based`` setting, like this:
         config:
           0:
             user: crate
+
+Authenticating to Admin UI
+==========================
+
+.. hide:
+
+    cr> CREATE USER admin;
+    CREATE OK, 1 row affected (... sec)
+
+When trying to access the Admin UI of CrateDB, initially an authentication with
+the user defined with the :ref:`auth.trust.http_default_user
+<auth_trust_http_default_user>` setting (defaults to ``crate``) will be
+attempted. If this authentication attempt fails, the browser will open the
+standard popup window where the user is asked to fill in credentials. Depending
+on the HBA configuration, it maybe not be necessary to provide a password but
```

Comment:
"it may be necessary to a username and password, or, alternatively, a username only"